requirements.txt

```diff
@@ -1,8 +1,8 @@
+django-terra-utils==0.3.0
```

Comment:
This package, and django-geostore shouldn't fixed to 0.3.x ?

Reply:
done